tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/BlockingMiddleware.cs

```diff
@@ -118,7 +118,7 @@ internal async Task Invoke(HttpContext context)
             }
             catch (Exception e) when (GetBlockException(e) is { } blockException)
             {
-                var action = security.GetBlockingAction(blockException!.Result.Actions[0], context.Request.Headers.GetCommaSeparatedValues("Accept"));
+                var action = security.GetBlockingAction(blockException!.Result!.Actions[0], context.Request.Headers.GetCommaSeparatedValues("Accept"));
```

Comment:
minor: are we sure result can never be null? If so couldn't we changes the definition so that that the `!` is not needed.

Reply:
Fixed in [3c11abb](https://github.com/DataDog/dd-trace-dotnet/pull/4942/commits/3c11abb2880c8cd81e70c7b5846ab05dd564e5e0)